deployments/charts/kuma/templates/cp-webhooks-and-secrets.yaml

```diff
@@ -297,3 +298,4 @@ webhooks:
           - gatewayclasses
     sideEffects: None
 {{- end }}
+{{- end }}
```

Comment:
This way you skip everything (hooks too), not just secrets. Is this intended?

Reply:
True, I think when you have no certs defined you cannot create hooks because they depends on `caBundle`